streaming/index.js

```diff
@@ -165,6 +166,68 @@ const redisConfigFromEnv = (env) => {
   };
 };
 
+const FALSE_VALUES = [
+  false,
+  0,
+  '0',
+  'f',
+  'F',
+  'false',
+  'FALSE',
+  'off',
+  'OFF',
+];
+
+/**
+ * @param {any} value
+ * @returns {boolean}
+ */
+const isTruthy = value =>
+  value && !FALSE_VALUES.includes(value);
```

Comment:
`const FALSE_VALUES = new Set([...])` and `FALSE_VALUES.has(value)` avoids iterating over the entire array.